odoo/addons/base/models/ir_translation.py

```diff
@@ -881,18 +898,18 @@ def get_translations_for_webclient(self, mods, lang):
                 "grouping": langs.grouping,
                 "decimal_point": langs.decimal_point,
                 "thousands_sep": langs.thousands_sep,
-                "week_start": langs.week_start,
+                "week_start": int(langs.week_start),
+                "code": lang,
             }
-            lang_params['week_start'] = int(lang_params['week_start'])
-            lang_params['code'] = lang
 
         # Regional languages (ll_CC) must inherit/override their parent lang (ll), but this is
         # done server-side when the language is loaded, so we only need to load the user's lang.
         translations_per_module = {}
         messages = self.env['ir.translation'].sudo().search_read([
             ('module', 'in', mods), ('lang', '=', lang),
-            ('comments', 'like', 'openerp-web'), ('value', '!=', False),
-            ('value', '!=', '')],
+            ('web', '=', True),
+            ('frontend', '=', True) if frontend else (1, '=', 1),
```

Comment:
A bit weird to generate a `1=1` clause, no?

Reply:
s'pretty common for dynamic domain, and it seemed more sensible than conditional concatenation or whatever.